Objects/object.c

```diff
@@ -276,8 +276,7 @@ PyObject_CallFinalizer(PyObject *self)
 
     /* The former could happen on heaptypes created from the C API, e.g.
        PyType_FromSpec(). */
```

Comment:
I think this comment (including the existence of a "former" depends on removed code, and should therefore itself be removed.